packages/invoices/src/InboundShipment/DetailView/SidePanel.tsx

```diff
@@ -102,6 +107,71 @@ const RelatedDocumentsSection: FC = () => {
   );
 };
 
+export const PricingSection = () => {
```

Comment:
Ugly - will address next in https://github.com/openmsupply/openmsupply-client/issues/572